apps/sourcegraph/server/app.yaml

```diff
@@ -1,5 +1,5 @@
 {
-    "id": "/sourcegraph",
+    "id": "/sourcegraph/server",
```

Comment:
Nice, this is definitely better, I was tempted to name it like this originally, but didn't end up doing it for simplicity. Anyway, I think this is a much better setup, especially if we want to add more code intelligence servers in the future.